core/variable_map.js

```diff
@@ -26,9 +26,6 @@
 
 goog.provide('Blockly.VariableMap');
 
-goog.require('Blockly.VariableModel');
```

Comment:
The require for Blockly.VariableModel is needed so this should be added back in.